lib/content-services/src/lib/dialogs/dialog/dialog.component.scss

```diff
@@ -0,0 +1,220 @@
+$theme-blue-button-color: #1f74db;
```

Comment:
Probaby you can use one of the defined theme colors? Instead of pinning it to the specific value?

Reply:
We don't have it in ADF but we have it ACA and I copy the name for consistency

Reply:
Use `color="primary"` for now. If designers prefer another color I'll create a story to update primary color